dff/messengers/telegram/__init__.py

```diff
@@ -1,2 +1,30 @@
 # -*- coding: utf-8 -*-
 # flake8: noqa: F401
+
+try:
+    import telebot
+except ImportError:
+    raise ImportError("telebot is not installed. Run `pip install dff[telegram]`")
+
+from .messenger import TelegramMessenger
+from .interface import PollingTelegramInterface, CallbackTelegramInterface
+from .message import TelegramUI, TelegramMessage, RemoveKeyboard
+from dff.script.core.message import Location, Attachment, Audio, Video, Image, Document, Attachments, Button, Command
```

Comment:
Why do we import `Location, Attachment, Audio, Video, Image, Document, Attachments, Button, Command` in this file. This for a user purpose if it is ,can user import from `dff.script.core.message` by yourself?

Reply:
Yes, I added those imports so that a user could import them with
```
from dff.messengers.telegram import Button, TelegramUI
```
instead of
```
from dff.messengers.telegram import TelegramUI
from dff.script.core.message import Button
```
.
Removed those imports in https://github.com/deeppavlov/dialog_flow_framework/pull/13/commits/50e66f31eb42c1addec56e31bff424262b866ca0.